user-console/src/main/java/org/pentaho/mantle/client/ui/tabs/MantleTabPanel.java

```diff
@@ -85,6 +156,16 @@ public void onWindowClosing( ClosingEvent event ) {
     } );
   }
 
+  public static void setTabsMenu( MenuBar menuBar, MenuItem menuItem ) {
+    tabsMenuBar = menuBar;
+    tabsMenuItem = menuItem;
+    tabsSubMenuBar = new TabContextMenuBar( true );
+    tabsSubMenuBar.addStyleName( "tabsSubMenuBar" );
+    tabsMenuItem.setSubMenu( tabsSubMenuBar );
+    tabsMenuBar.addStyleName("flex-row" );
```

Comment:
suggestion: add `"flex-row"` to variable at top of class

Reply:
> I also suggest moving it to a CSS file, if possible.
I agree that is a better solution, similar to my later comment
https://github.com/pentaho/pentaho-platform/pull/5350#discussion_r1316474847

Reply:
How do I add a class to a dom node via css file?

Reply:
yeah, I thought the point of creating these global css classes was so that they were re-usable